spacelift/resource_stack_test.go

```diff
@@ -493,6 +493,7 @@ func TestStackResource(t *testing.T) {
 					Attribute("id", StartsWith("provider-test-stack")),
 					Attribute("kubernetes.0.namespace", Equals("")),
 					Attribute("kubernetes.0.kubectl_version", IsNotEmpty()),
+					Attribute("kubernetes.0.kubernetes_workflow_version", Equals("KUBERNETES")),
```

Comment:
shouldn't this be `kubernetes_workflow_tool` ?

Reply:
fixewd